File: src/main/java/io/github/oleksivio/telegram/bot/api/model/objects/inline/queryresult/InlineQueryResultGame.java
package io.github.oleksivio.telegram.bot.api.model.objects.inline.queryresult;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * @see <a href="https://core.telegram.org/bots/api#inlinequeryresultgame">InlineQueryResultGame</a>
 */
public class InlineQueryResultGame extends InlineQueryResult {
    private static final String TYPE = "game";
    /**
     * game_short_name String Short name of the game
     */
    @JsonProperty("game_short_name")
    private final String gameShortName = null;

    InlineQueryResultGame() {
        super(TYPE);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/objects/inline/queryresult/InlineQueryResultGame.kt
package io.github.oleksivio.telegram.bot.api.model.objects.inline.queryresult

import com.fasterxml.jackson.annotation.JsonProperty

/**
 *  [InlineQueryResultGame](https://core.telegram.org/bots/api/#inlinequeryresultgame)
 */
class InlineQueryResultGame(
        /**
         * game_short_name String Short name of the game
         */
        @JsonProperty("game_short_name")
        val gameShortName: String,
        /**
         * type String Type of the result
         */
        @JsonProperty("type")
        val type: String = "game"
) : InlineQueryResult()
